core/amber/src/main/scala/edu/uci/ics/texera/workflow/common/storage/OpResultStorage.scala

```diff
@@ -2,17 +2,24 @@ package edu.uci.ics.texera.workflow.common.storage
 
 import java.util.concurrent.ConcurrentHashMap
 import com.typesafe.scalalogging.LazyLogging
+import edu.uci.ics.amber.engine.common.AmberUtils
 import edu.uci.ics.texera.workflow.common.tuple.schema.Schema
 import edu.uci.ics.texera.workflow.operators.sink.storage.{
   MemoryStorage,
   MongoDBStorage,
   SinkStorageReader
 }
 
+object OpResultStorage {
+  val defaultStorageMode: String = AmberUtils.amberConfig.getString("storage.mode").toLowerCase
+  val MEMORY = "memory"
+  val MONGODB = "mongodb"
+}
+
 /**
   * Public class of operator result storage.
   */
-class OpResultStorage(mode: String = "memory") extends Serializable with LazyLogging {
+class OpResultStorage extends Serializable with LazyLogging {
```

Comment:
I think this whole thing can be moved to `object` and it can be globally accessible without creating an instance

Reply:
added some comments to explain the lifecycle of this class. We keep the existing code.